File: programmers/코딩테스트 입문 with Java/외계행성의나이.java
/**
 * https://school.programmers.co.kr/learn/courses/30/lessons/120834
 */
public class 외계행성의나이 {
    public static void main(String[] args) {
        System.out.println(solution(23));
        System.out.println(solution(51));
        System.out.println(solution(100));
    }

    public static String solution(int age) {
        String answer = "";
        String str = Integer.toString(age);
        for (int i = 0; i < str.length(); i++) {
            answer+=(char)(str.charAt(i)+49);
        }
        return answer;
    }
}

File: programmers/코딩테스트 입문 with Python/외계행성의 나이.py
def solution(age):
    language = {0: 'a', 1: 'b', 2: 'c', 3: 'd', 4: 'e', 5: 'f', 6: 'g', 7: 'h', 8: 'i', 9: 'j'}
    answer = ''
    while age // 10 != 0:
        answer += language[age % 10]
        age //= 10
        print(answer)
    answer += language[age % 10]
    return answer[::-1]


print(solution(23))
print(solution(51))
print(solution(100))
print(solution(9))
